kubernetes/build-cam-database.yaml

```diff
@@ -14,7 +13,7 @@ spec:
         spec:
           containers:
           - name: pipeline-tools
-            image: "renciorg/cam-pipeline-tools:v1.3"
+            image: "renciorg/cam-pipeline-tools:dev"
```

Comment:
I don't see the benefit of using a tag whose contents can change for docker images. It seems like you want to be able to rely on the environment being immutable.

Reply:
I'm okay with accepting that, but I always like tying things to specific versions -- that way, if something changes in the future, we know exactly which version was the last one to work. (Another reason I like fixed Maven versions better than "must be compatible with" versions in Python package management.)

Reply:
Oh wait, you suggested the exact opposite of what I just said! Please ignore my previous comment :). Replaced `dev` with `v1.4` in 05e0e92.

Reply:
Sounds like we're in violent agreement. :-)